src/Workload/Microsoft.NET.Sdk.Maui.Manifest/WorkloadManifest.in.json

```diff
@@ -138,6 +139,10 @@
         "any": "Microsoft.Maui.Sdk"
       }
     },
+    "Microsoft.NET.Sdk.Maui": {
+      "kind": "sdk",
+      "version": "@VERSION@"
+    },
```

Comment:
This is the magic that makes things work - no version here as we are using the workload version.